tool/tctl/common/collection.go

```diff
@@ -1003,6 +1005,34 @@ func (c *integrationCollection) writeText(w io.Writer, verbose bool) error {
 	return trace.Wrap(err)
 }
 
+type externalCloudAuditCollection struct {
+	externalCloudAudits []*externalcloudaudit.ExternalCloudAudit
+}
+
+func (c *externalCloudAuditCollection) resources() (r []types.Resource) {
+	for _, a := range c.externalCloudAudits {
+		r = append(r, a)
+	}
+	return r
+}
+
+func (c *externalCloudAuditCollection) writeText(w io.Writer, verbose bool) error {
+	var rows [][]string
+	for _, a := range c.externalCloudAudits {
+		specProps := []string{}
```

Comment:
It looks like `specProps` doesn't need to be an array. It never contains more than one value.

Reply:
reworked in https://github.com/gravitational/teleport/pull/32833/commits/322c201c565bcc00d2b4a8e229eb09d06656ad84